### ISSUE TITLE
!important can cause merging to behave strangely

### ISSUE THREAD

**Issue by mr-beerkiss** [jakubpawlowicz/clean-css#675]
If you have two selectors which define a property with the latter (in order of which it will appear in the source file) being marked as `!important` that property will be completely removed from the merge.

A simple input file with two selectors is enough (I say selectors but I've only tested with classes thus far).  I've included all the combinations I have tried and included their output.  I've used `display` but it appears to affect other properties as well.

```
/* !important set on latter selector, output is blank file */
.hidden {
  display: hidden;
}

.hidden {
  display: hidden !important;
}

/* !important applied to former selector, output normal */

.hidden {
   display: hidden !important;
}

.hidden { 
   display: hidden;
}

/* No use of !important, output normal */
.hidden {
  display: hidden;
}

.hidden {
  display: hidden;
}

/* All selectors use !important, output normal */
.hidden {
  display: none !important;
}

.hidden {
  display: none !important;
}

/* !important appears in the middle of congruently named selectors, 
   output is defined by last selector */
.hidden {
    display: none;
}

.hidden {
    display: !important;
}

.hidden {
    display: none;  /* this will appear in the output file */
}
```

**Comment by mr-beerkiss** [comment]
Update: running the with `--skip-advanced` option will solve the issue.  This may appear obvious but it was noted separately in #680 so I thought it best to be verbose.

**Comment by jakubpawlowicz** [comment]
Thanks for reporting this @mr-beerkiss. I'll push the fix tomorrow.

**Comment by jakubpawlowicz** [comment]
Fixed in 3.4.5.
